docs/VerifPlans/Simulation/micro_architecture/CV32E40X_OBI_VerifPlan.csv

```diff
@@ -0,0 +1,82 @@
+Requirement Location,Feature,Sub Feature,Feature Description,Verification Goal,Pass/Fail Criteria,Test Type,Coverage Method,Link to Coverage,Comments
```

Comment:
Is this jus X vplan? Not a SX vplan?

Reply:
I though at least the PMA vplan was a XS vplan. But maybe this will be fixed in a merge or so?

Reply:
Those look wong indeed.

I don't think this PR is the best place to fix it, but I've created a task to continue this.

Reply:
Okei, good!